Installing fresh template component won't compile

Hi! Thanks for those templates, it makes super easy to get start with ratatui.

When I ran: `cargo generate ratatui/templates` and picked `component` it created project, but running `cargo build` fails:

```
error[E0277]: the trait bound vergen::feature::build::Build:
   vergen_lib::entries::Add is not satisfied
     --> build.rs:11:23
      |
   11 |     .add_instructions(&build)?
      |                       ^^^^^^ the trait
  vergen_lib::entries::Add is not implemented for
  vergen::feature::build::Build
      |
  note: there are multiple different versions of crate
  vergen_lib in the dependency graph
```

Not sure if that a 'me' problem with my dependencies, or might happen for someone else, but fix for me was to update `vergen-gix` to `9.1.0` in `Cargo.toml` and than run `cargo build` again, now build passes and ui works

```
cargo 1.93.0 (083ac5135 2025-12-15)
rustc 1.93.0 (254b59607 2026-01-19)
```

--- a/component-generated/Cargo.toml
+++ b/component-generated/Cargo.toml
@@ -43,7 +43,7 @@ tracing-subscriber = { version = "0.3.18", features = ["env-filter", "serde"] }
 
 [build-dependencies]
 anyhow = "1.0.90"
-vergen-gix = { version = "1.0.2", features = ["build", "cargo"] }
+vergen-gix = { version = "9.1.0", features = ["build", "cargo"] }
 
 # Read the optimization guideline for more details: https://ratatui.rs/recipes/apps/release-your-app/#optimizations
 [profile.release]

--- a/component/template/Cargo.toml
+++ b/component/template/Cargo.toml
@@ -45,7 +45,7 @@ tracing-subscriber = { version = "0.3.18", features = ["env-filter", "serde"] }
 
 [build-dependencies]
 anyhow = "1.0.90"
-vergen-gix = { version = "1.0.2", features = ["build", "cargo"] }
+vergen-gix = { version = "9.1.0", features = ["build", "cargo"] }
 
 # Read the optimization guideline for more details: https://ratatui.rs/recipes/apps/release-your-app/#optimizations
 [profile.release]

